src/pages/workspaces/workspace/analysis/modals/ComputeModal.test.js

```diff
@@ -428,8 +427,7 @@ describe('ComputeModal', () => {
     })
 
     // Assert
-    const headerForNextPane = await screen.findByText('Downtime required')
-    expect(headerForNextPane).toBeInTheDocument()
+    screen.findByText('Downtime required')
```

Comment:
`findBy` queries return a Promise and should be awaited. Alternatively, could this use `queryBy` instead of `findBy`.

https://testing-library.com/docs/queries/about/#types-of-queries